lib/CXGN/BrAPI/v2/ObservationVariables.pm

```diff
@@ -260,6 +260,21 @@ sub get_query {
     $sth->execute();
     while (my ($cvterm_id, $cvterm_name, $cvterm_definition, $db_name, $db_id, $db_url, $dbxref_id, $accession, $synonym, $obsolete, $additional_info_string, $count) = $sth->fetchrow_array()) {
         $total_count = $count;
+        foreach (@$synonym){
+            $_ =~ s/ EXACT \[\]//;
+            $_ =~ s/\"//g;
+        }
+
+        my $trait = CXGN::Trait->new({bcs_schema=>$self->bcs_schema, cvterm_id=>$cvterm_id});
+        my $categories = $trait->categories;
+        my @categories = split '/', $categories;
+        my @brapi_categories;
+        foreach (@categories) {
+            push @brapi_categories, {
+                label => $_,
+                value => $_
+            };
+        }
```

Comment:
Do you know if the synonyms and categories code above was something added by their code coming in? The variables they are parsing those into aren't used anywhere. 

No need to fix it if it was part of the code. I more so want to check that you didn't add this when debugging stuff and it got left behind.

Reply:
@ctucker3  That's an incoming change.  I tried to make merge commits that I only made changes for conflicts.  Any other changes I made were in separate commits (and I didn't touch this file).

Reply:
Weird! Well it doesn't do any harm so I'm fine with leaving it there.